helmfile.d/values/admin-namespaces-sc.yaml.gotmpl

```diff
@@ -1,9 +1,11 @@
 namespaces:
+  {{- if not .Values.clusterApi.enabled }}
   - name: cert-manager
     labels:
       pod-security.kubernetes.io/audit: restricted
       pod-security.kubernetes.io/enforce: restricted
       pod-security.kubernetes.io/warn: restricted
+  {{- end }}
```

Comment:
Instead of this could we ensure that the namespace gets the required annotations and labels to be adopted into this release?

Reply:
Do you know of a way of doing this without having to invoke other commands before `helmfile apply`? It would be nice to at some point only be helmfile dependent so adding more steps here would suck. I guess we could label and annotate at the cluster provisioning step if there is no other option?

Reply:
Yes, I'm thinking at the cluster provisioning step as it would be something that we'd have to do special once rather than all the time.

I guess we could try a hook in the namespace helm chart to set such annotations and labels but I'm unsure if it would check that before and abort anyway. :thinking:

Reply:
https://github.com/elastisys/ck8s-cluster-api/pull/160 is merged now, PTAL at the cleanup script changes :)